modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -3,11 +3,9 @@ import {Meta, ArgsTable} from '@storybook/addon-docs';
 import {Grid, GridItem} from '@workday/canvas-kit-react/layout';
 // examples
 import {GridLayout} from './examples/Grid/GridLayout';
-import {UIExample} from './examples/Grid/UIExample';
+import {Basic} from './examples/Grid/Basic';
 import {GridLayoutInteractive} from './examples/Grid/GridLayoutInteractive';
-import {GridStyle} from './examples/PropTables.splitprops.tsx';
-import {GridStyleProps} from '../lib/utils/grid';
-import {GridItemStyleProps} from '../lib/utils/gridItem';
+import {GridStyle, GridItemStyle} from './examples/PropTables.splitprops.tsx';
```

Comment:
I adjusted these imports to import both `Grid` and `Grid.Item` prop table components from the layout `PropTables.splitprops.tsx` instead of `lib/utils/grid` and `lib/utils/gridItem` (which didn't work because those files aren't named `PropTables.splitprops.tsx` and thus have all their props filtered out since they're layout components).

Reply:
Thank you!